Mlem/App/Views/Pages/ExternalApiInfoView.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  ExternalApiInfoView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 08/06/2024.
+//
+
+import MlemMiddleware
+import SwiftUI
+
+struct ExternalApiInfoView: View {
+    @Environment(AppState.self) private var appState
+    @Environment(Palette.self) private var palette
+    
+    @State private var isLoading: Bool = true
+    @State private var internalFederationStatus: FederationStatus?
+    @State private var externalFederationStatus: FederationStatus?
+    @State private var externalInstance: Instance3?
+    
+    /// The ``ApiClient`` of the model being inspected.
+    let fallbackApi: ApiClient
+    /// The `host` of the model being inspected.
+    let entityActorId: URL
+    /// The local ``ApiClient`` of the model, created using `entityHost`.
+    let entityLocalApi: ApiClient
+    
+    init(api: ApiClient, actorId: URL) {
+        self.fallbackApi = api
+        self.entityActorId = actorId
+        self.entityLocalApi = .getApiClient(for: actorId.removingPathComponents(), with: nil)
+    }
+    
+    var body: some View {
+        VStack {
+            if isLoading {
+                Text("Diagnosing...")
+                    .foregroundStyle(.secondary)
+            } else {
+                ScrollView {
+                    content
+                }
+            }
+        }
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .animation(.easeOut(duration: 0.2), value: isLoading)
+        .background(palette.groupedBackground)
+        .task(loadData)
+        .presentationDetents([.medium])
+        .presentationBackgroundInteraction(.enabled)
+    }
+    
+    @ViewBuilder
+    var content: some View {
+        VStack(spacing: 16) {
+            box(spacing: 16) {
+                if internalFederationStatus?.isAllowed ?? false, externalFederationStatus?.isAllowed ?? false {
+                    Text(
+                        // swiftlint:disable:next line_length
+                        "Your instance and **\(entityLocalApi.host ?? "")** appear to be federating with one another, but we weren't able to access this content from your instance. This could be because the content is new, and hasn't federated to your instance yet. Or, it could be because your instance has purged this content."
```

Comment:
```suggestion
                        "Your instance and **\(entityLocalApi.host ?? "")** federate, but the content could not be loaded. It may not have federated yet, or your instance may have purged it."
```

We don't *technically* speaking know for 100% certain that they federate, but we're sure enough that it's not worth the extra words imo